.github/workflows/analyze-and-build.yml

```diff
@@ -62,8 +62,8 @@ jobs:
       - uses: actions/setup-node@master
         with: {node-version: ^12.0}
       - uses: actions/checkout@master
+      - run: gem install bundler:1.17.2 && bundle check || bundle install --frozen --path vendor/bundle
```

Comment:
Why was this necessary? CocoaPods' version requirement on bundler of `~> 1.3`?

Reply:
If I don’t specify a version, then whatever `gem` wound up installing didn’t work, and the error message suggested this.

Reply:
Ah okay, that clears things up.